frontend/src/lib/rest/events.js

```diff
@@ -23,6 +24,14 @@ export default {
   async delete(id) {
     try {
       await restClient.delete(`/events/${id}`)
+      const res = await restClient.get('/invalid-events', {})
```

Comment:
Ist this still needed because you can add an SQL constraint, that deletes the referenced invalid events automatically.

With the following constraint in the `invalid_events` table:

```
event_id uuid NOT NULL REFERENCES events(id) ON DELETE CASCADE ON UPDATE CASCADE
```